ntp-udp/src/socket.rs

```diff
@@ -352,26 +354,29 @@ fn fetch_send_timestamp_help(
                 // the timestamping does not set a message; if there is a message, that means
                 // something else is wrong, and we want to know about it.
                 if error.ee_errno as libc::c_int != libc::ENOMSG {
-                    warn!(
+                    tracing::warn!(
                         expected_counter,
-                        error.ee_data, "error message on the MSG_ERRQUEUE"
+                        error.ee_data,
+                        "error message on the MSG_ERRQUEUE"
                     );
                 }
 
                 // Check that this message belongs to the send we are interested in
                 if error.ee_data != expected_counter {
-                    warn!(
+                    tracing::info!(
                         error.ee_data,
-                        expected_counter, "Timestamp for unrelated packet"
+                        expected_counter,
+                        "Timestamp for unrelated packet"
```

Comment:
I think this should just be debug. The actual problem is that we didn't get the timestamp at the point we expected it, and we should log it as a warning there.